site/_docs/reference.md

```diff
@@ -2653,11 +2653,14 @@ BigQuery's type system uses confusingly different names for types and functions:
 | * | ACOSH(numeric)                                 | Returns the inverse hyperbolic cosine of *numeric*
 | s | ARRAY(expr [, expr ]*)                         | Construct an array in Apache Spark
 | b | ARRAY_CONCAT(array [, array ]*)                | Concatenates one or more arrays. If any input argument is `NULL` the function returns `NULL`
-| s | ARRAY_DISTINCT(array)                          | Returns unique elements of *array*. Keeps ordering of elements.
+| s | ARRAY_DISTINCT(array)                          | Returns unique elements of *array*. Keeps ordering of elements
```

Comment:
Returns unique elements that keep ordering of elements of *array*

Reply:
Maybe `Removes duplicate values from the *array*`

https://spark.apache.org/docs/latest/api/sql/#array_distinct

Reply:
@JiajunBernoulli fixed @JiajunBernoulli

Reply:
In fact, Keeps ordering of elements is a sentence that I'm worried about.

This sentence lacks a subject, I hope the description is close to natural language.
For example:
Removes duplicate values from the *array* that keeps ordering of elements.

Reply:
thanks for your suggestion, i fxied @JiajunBernoulli